helper/helper/src/main/java/com/linkedin/avroutil1/compatibility/RecordGenerationConfig.java

```diff
@@ -38,26 +41,39 @@ public Random random() {
         return randomToUse;
     }
 
+    public boolean avoidNulls() {
```

Comment:
Can this have a more intuitive name? Or some JavaDoc to explain the intent? Is it to avoid a "null-only" type? Or to avoid nulls in unions? Or some other thing?

Reply:
added javadoc, both to the method and fields